Tools/Release/updateAMReX.py

```diff
@@ -123,9 +123,9 @@
 with open(ci_gnumake_path, encoding='utf-8') as f:
     ci_gnumake_content = f.read()
     #   branch/commit/tag (git fetcher) version
-    #     cd amrex && git checkout COMMIT_TAG_OR_BRANCH && cd -
+    #     cd ../amrex && git checkout COMMIT_TAG_OR_BRANCH && cd -
     ci_gnumake_content = re.sub(
-        r'(.*cd\s+amrex.+git checkout\s+--detach\s+)(.+)(\s+&&\s.*)',
+        r'(.*cd\s+../amrex.+git checkout\s+--detach\s+)(.+)(\s+&&\s.*)',
```

Comment:
Technically, the `.` will match any character since this is a regex.
Explicit would be
```suggestion
        r'(.*cd\s+\.\./amrex.+git checkout\s+--detach\s+)(.+)(\s+&&\s.*)',
```
to escape a literal `.`

Python regex special characters are: `.`, `+`, `*`, `?`, `^`, `$`, `(`, `)`, `[`, `]`, `{`, `}`, `|`, `\`